lib/l10n/app_en.arb

```diff
@@ -231,29 +231,29 @@
 
   "albumTierNew": "New",
   "albumTierPinned": "Pinned",
-  "albumTierSpecial": "Common",
+  "albumTierSpecial": "Often used",
   "albumTierApps": "Apps",
   "albumTierRegular": "Others",
 
   "storageVolumeDescriptionFallbackPrimary": "Internal storage",
   "storageVolumeDescriptionFallbackNonPrimary": "SD card",
-  "rootDirectoryDescription": "root directory",
-  "otherDirectoryDescription": "“{name}” directory",
+  "rootDirectoryDescription": "root folder",
+  "otherDirectoryDescription": "“{name}” folder",
   "@otherDirectoryDescription": {
     "placeholders": {
       "name": {
         "type": "String",
         "example": "Pictures",
-        "description": "the name of a specific directory"
+        "description": "the name of a specific folder"
       }
     }
   },
-  "storageAccessDialogMessage": "Please select the {directory} of “{volume}” in the next screen to give this app access to it.",
+  "storageAccessDialogMessage": "Please select the {directory} of “{volume}” next to give this app access to it.",
   "@storageAccessDialogMessage": {
     "placeholders": {
       "directory": {
         "type": "String",
-        "description": "the name of a directory, using the output of `rootDirectoryDescription` or `otherDirectoryDescription`"
+        "description": "the full name of a folder, using the output of `rootDirectoryDescription` or `otherDirectoryDescription`"
```

Comment:
A bit awkward for experienced users, but this allows doing away with "directory" entirely.